js/sparkswap.js

```diff
@@ -372,7 +372,68 @@ module.exports = class sparkswap extends Exchange {
     }
 
     async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
-        throw new ExchangeError ('Not Implemented');
+        const CONSTANTS = {
+            'STATUSES': {
+                'ACTIVE': 'ACTIVE',
+                'CANCELLED': 'CANCELLED',
+                'COMPLETED': 'COMPLETED',
+                'FAILED': 'FAILED',
+            },
+            'SIDES': {
+                'ASK': 'ASK',
+                'BID': 'BID',
+                'BUY': 'buy',
+                'SELL': 'sell',
+            },
+            'TYPES': {
+                'LIMIT': 'limit',
+                'MARKET': 'market',
+            },
+        };
+        const res = await this.privateGetV1Orders ({ 'market': symbol });
```

Comment:
Do you want to follow the same pattern of:
```
await this.loadMarkets ();

const res = await this.privateGetV1Orders ({ 'market': this.marketId (symbol) });
```

Reply:
yes! good catch @martineehrlich